webextensions/api/webRequest.json

```
@@ -117,18 +117,26 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/RequestFilter",
             "support": {
               "chrome": {
+                "notes": "If a filter contains unrecognized values in its <code>types</code> property, <code>addListener()</code> will throw an exception.",
```

It's sort of an unwritten rule (see #6229) but I've been steering people toward all notes being written in the present tense (for reviewability and maintainability)

```suggestion
                "notes": "If a filter contains unrecognized values in its <code>types</code> property, <code>addListener()</code> throws an exception.",
```